### PR TITLE
Fix: three not found view v2

### PR DESCRIPTION
**Pull request by kosmydel**
### Details

This is the second version, after the first one PR #36409 was reverted due to the #40509 issue.

This PR addresses three things:
- fixes multiple not-found views,
- fixes the back button,
- refactors the `WorkspaceMembersPage` to use the `WorkspacePageWithSections` component.

### Fixed Issues

$ https://github.com/Expensify/App/issues/34234
$ https://github.com/Expensify/App/issues/36266
$ https://github.com/Expensify/App/issues/40509
PROPOSAL: N/A

### Tests

1. Do tests from the QA Steps section.
2. You can test the 6th QA step using this command: `adb shell am start -a android.intent.action.VIEW -d "new-expensify://settings/workspaces/kjashdjkashdaks/rateandunit"`

### Offline tests

N/A

### QA Steps

1. Verify that no errors appear in the JS console
2. Verify that navigation works correctly.
3. Verify the deleting workspace flow:
  -  [Main device] Create a new Workspace
  -  [Main device] Navigate to Members
  -  [Secondary device] Delete Workspace created in the Main device
  -  [Main device] Click on "Go back to home page" in LHN
4. On the Web (multiple not found views):
  -  Go to the All Settings
  -  Select a workspace
  -  Change the workspace ID in the URL to a nonexistent one (e.g. `https://staging.new.expensify.com/settings/workspaces/kjashdjkashdaks/profile`)
  -  Verify that we have only one not-found view.
  -  Verify that we can go back.
  -  Verify other links:
      -  https://staging.new.expensify.com/settings/workspaces/kjashdjkashdaks/invoices
      -  https://staging.new.expensify.com/settings/workspaces/kjashdjkashdaks/reimburse
  -  With RHP modal opened: 
      -  https://staging.new.expensify.com/settings/workspaces/kjashdjkashdaks/profile/name
      -  https://staging.new.expensify.com/settings/workspaces/kjashdjkashdaks/rateandunit
      -  https://staging.new.expensify.com/settings/workspaces/kjashdjkashdaks/rateandunit/unit
      -  https://staging.new.expensify.com/settings/workspaces/kjashdjkashdaks/invite
      -  https://staging.new.expensify.com/settings/workspaces/kjashdjkashdaks/invite-message
      -  https://staging.new.expensify.com/settings/workspaces/341F422BC2DD838x/more-features
      - https://staging.new.expensify.com/settings/workspaces/341F422BC2DD838x/more-features
      - https://staging.new.expensify.com/settings/workspaces/341F422BC2DD838x/tags
      - https://staging.new.expensify.com/settings/workspaces/341F422BC2DD838x/taxes
      - https://staging.new.expensify.com/settings/workspaces/341F422BC2DD838x/distance-rates
      -  Existing, without permissions: https://staging.new.expensify.com/settings/workspaces/E2BFF0D3DDCB1520/invite
   - Try also other links.
5. Paste the links from step 4 in the chat, and click links, to verify that in-app deep links are working correctly.
6. Verify the regression:
    1. Open the Android app
    2. Create a workspace
    3. Go to workspace settings page
    4. From another app deep link to any of these links:
      - https://staging.new.expensify.com/settings/workspaces/kjashdjkashdaks/rateandunit
      - https://staging.new.expensify.com/settings/workspaces/kjashdjkashdaks/rateandunit/unit
      - https://staging.new.expensify.com/settings/workspaces/kjashdjkashdaks/invite
      - https://staging.new.expensify.com/settings/workspaces/kjashdjkashdaks/invite-message
    5. Try also other links.
7. Check if #38977 is not broken:
    - Open the New Expensify app.
    - Login
    - Popup appears (or type `Onyx.set('focusModeNotification', true);` in console)
    - Click on the text link “Settings”
    - Verify that the Preferences page is not visible and priority mode page opens
8. **Test everything without strict mode**. Go to `CONFIG.ts` and set `CONFIG.USE_REACT_STRICT_MODE` to `false`.

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.

### Screenshots/Videos

| Test case | Video |
|--------|--------|
| 3 | https://github.com/Expensify/App/assets/104823336/22982f76-c8cb-4000-8283-245c59d504e4 |
| 4, 5 | https://github.com/Expensify/App/assets/104823336/4c3724c8-a9b9-4b33-aac5-eee82eb3e64c |
| 6 | https://github.com/Expensify/App/assets/104823336/14ae7a66-380f-4e09-82bc-8be3cba84c84 |
| 7 | https://github.com/Expensify/App/assets/104823336/705636b8-b32a-47a8-9755-67b30c5940b3 | 

<details>
<summary>Android: Native</summary>

https://github.com/Expensify/App/assets/104823336/222afc6b-7298-4c5a-ac03-38f19b3bb300

</details>

<details>
<summary>iOS: Native</summary>

</details>

<details>
<summary>iOS: mWeb Safari & Android: mWeb Chrome</summary>

https://github.com/Expensify/App/assets/104823336/b85cb23b-159a-4b77-a602-c0d1bc25cf4d

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

https://github.com/Expensify/App/assets/104823336/33f222b5-1f6b-42d4-920f-53fd41d76173

</details>

<details>
<summary>MacOS: Desktop</summary>

https://github.com/Expensify/App/assets/104823336/d84c5b3f-01f2-4733-a7cb-c4c81536b437

</details>

### CODE REVIEW

File: src/libs/Navigation/dismissRHP.ts
Comment (WojtekBoman) [inline src/libs/Navigation/dismissRHP.ts:11]: Do we actually pass this param?

File: src/libs/Navigation/linkingConfig/getAdaptedStateFromPath.ts
Comment (WojtekBoman) [inline src/libs/Navigation/linkingConfig/getAdaptedStateFromPath.ts:18]: Why has `as const` been removed?

**Review by WojtekBoman — APPROVED** [review]

File: src/libs/Navigation/linkingConfig/getAdaptedStateFromPath.ts
Comment (kosmydel) [reply src/libs/Navigation/linkingConfig/getAdaptedStateFromPath.ts:18]: I wanted to narrow down the typing, as currently, we don't enforce the list items to be of `Screen` type. I think we can go with the following:
```javascript
const RHP_SCREENS_OPENED_FROM_LHN = [...] satisfies Screen[];
``` 

In this case we:
- enforce the list items to be of `Screen` type
- narrow down the typing (similarly to `as const`).

If this approach doesn't work, I can switch back to the original one.

WDYT?

Comment (WojtekBoman) [reply src/libs/Navigation/linkingConfig/getAdaptedStateFromPath.ts:18]: Sounds fine, thanks for the explanation :)

**Comment by kosmydel** [comment]
I believe that @situchan should be assigned here, as he was C+ of the original PR. I'm not sure if he is still OOO.

**Comment by shawnborton** [comment]
Screenshots look good to me

**Comment by situchan** [comment]
I am back. There's conflict

**Comment by kosmydel** [comment]
> I am back. There's conflict

I've resolved the conflict :)

**Comment by situchan** [comment]
```diff
-   }, [canAccessRoute, policy, shouldShowNotFoundPage]);
+   }, [canAccessRoute, shouldShowNotFoundPage]);
```

Is this the main change to fix regression of https://github.com/Expensify/App/issues/40509?

**Comment by kosmydel** [comment]
> Is this the main change to fix regression of #40509?

As far as I remember, yes. We shouldn't call the `useEffect` on each policy change. I'm not sure how the `policy` dependency got there initially.

**Comment by situchan** [comment]
Please fix conflict

**Comment by kosmydel** [comment]
> Please fix conflict

Done ✅

**Comment by situchan** [comment]
## Reviewer Checklist

- [ ] I have verified the author checklist is complete (all boxes are checked off).
- [ ] I verified the correct issue is linked in the `### Fixed Issues` section above
- [ ] I verified testing steps are clear and they cover the changes made in this PR
    - [ ] I verified the steps for local testing are in the `Tests` section
    - [ ] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [ ] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [ ] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [ ] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] Android: Native
    - [ ] Android: mWeb Chrome
    - [ ] iOS: Native
    - [ ] iOS: mWeb Safari
    - [ ] MacOS: Chrome / Safari
    - [ ] MacOS: Desktop
- [ ] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [ ] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [ ] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [ ] I verified that comments were added to code that is not self explanatory
    - [ ] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [ ] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [ ] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [ ] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [ ] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [ ] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [ ] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [ ] If a new component is created I verified that:
    - [ ] A similar component doesn't exist in the codebase
    - [ ] All props are defined accurately and each prop has a `/** comment above it */`
    - [ ] The file is named correctly
    - [ ] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [ ] The only data being stored in the state is data necessary for rendering and nothing else
    - [ ] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [ ] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [ ] All JSX used for rendering exists in the render method
    - [ ] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [ ] If any new file was added I verified that:
    - [ ] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [ ] If a new CSS style is added I verified that:
    - [ ] A similar style doesn't already exist
    - [ ] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG`)
- [ ] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [ ] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [ ] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [ ] I verified that all the inputs inside a form are aligned with each other.
    - [ ] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [ ] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [ ] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [ ] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>



</details>

<details>
<summary>Android: mWeb Chrome</summary>



</details>

<details>
<summary>iOS: Native</summary>



</details>

<details>
<summary>iOS: mWeb Safari</summary>



</details>

<details>
<summary>MacOS: Chrome / Safari</summary>



</details>

<details>
<summary>MacOS: Desktop</summary>



</details>

**Comment by situchan** [comment]
Another conflict

**Comment by kosmydel** [comment]
> Another conflict

Resolved ✅ 

Please double-check the `src/pages/workspace/WorkspaceMembersPage.tsx`, as there have been many conflicts over time.

File: src/libs/Navigation/linkingConfig/getAdaptedStateFromPath.ts
Comment (situchan) [inline src/libs/Navigation/linkingConfig/getAdaptedStateFromPath.ts:171]: Please make sure that this removal doesn't cause any regressions. Find out why this was added at the beginning

**Review by situchan — COMMENTED** [review]
Looks good

File: src/libs/Navigation/linkingConfig/getAdaptedStateFromPath.ts
Comment (kosmydel) [reply src/libs/Navigation/linkingConfig/getAdaptedStateFromPath.ts:171]: It was added in [this PR](https://github.com/Expensify/App/pull/38977). The test plan covers this issue (point 6). As far as I remember, I've added the priority mode screen to the `RHP_SCREENS_OPENED_FROM_LHN` to keep it working.

**Comment by kosmydel** [comment]
@situchan friendly bump :)

**Review by situchan — APPROVED** [review]
Looks good.
Please merge main, and then I will retest

**Comment by kosmydel** [comment]
> Looks good. Please merge main, and then I will retest

I've merged the main, thank you

**Review by situchan — APPROVED** [review]

**Review by neil-marcellini — CHANGES_REQUESTED** [review]
Your fixes on top of the original PR seem good. The testing is the most important thing. @situchan it looks like you only ran some of the tests. Would you please add videos showing the other tests like verifying the delete workspace flow and the focus mode settings link?

**Comment by kosmydel** [comment]
@neil-marcellini I agree. This PR requires a lot of testing, as it was already reverted.

**Comment by situchan** [comment]
Sure thing!
@kosmydel can you also please retest and update screen recordings?

**Comment by kosmydel** [comment]
| Test case | Video |
|--------|--------|
| 3 | https://github.com/Expensify/App/assets/104823336/22982f76-c8cb-4000-8283-245c59d504e4 |
| 4, 5 | https://github.com/Expensify/App/assets/104823336/4c3724c8-a9b9-4b33-aac5-eee82eb3e64c |
| 6 | https://github.com/Expensify/App/assets/104823336/14ae7a66-380f-4e09-82bc-8be3cba84c84 |
| 7 | https://github.com/Expensify/App/assets/104823336/705636b8-b32a-47a8-9755-67b30c5940b3 |

**Comment by neil-marcellini** [comment]
Bump @situchan

**Comment by kosmydel** [comment]
I've pulled the main to keep it up-to-date. 
Friendly bump @situchan :)

**Comment by situchan** [comment]
Thanks. will be done today

**Comment by situchan** [comment]
Faced issue with android deep link on my emulator. Will fix asap
Other scenarios tested well

**Comment by kosmydel** [comment]
> Faced issue with android deep link on my emulator. Will fix asap Other scenarios tested well

I've merged the main to keep it up-to-date. I will test it later today.

**Comment by situchan** [comment]
How did the test go?

Make sure to apply change from https://github.com/Expensify/App/pull/43893 while fixing conflict.

**Review by situchan — APPROVED** [review]
Added videos for each case in [checklist](https://github.com/Expensify/App/pull/41665#issuecomment-2136703867)

:shipit: from me

Let's just fix conflict.

**Comment by kosmydel** [comment]
@situchan After pulling the main internal deep links are broken on this branch. I'm going to investigate it. 

Next time please prioritize this PR, as this is taking a long time. Each time I pull the main I have to resolve conflicts, test it, and fix problems if appear. It is time-consuming.

**Comment by kosmydel** [comment]
I didn't have much time today, but this PR may conflict with [this one](https://github.com/Expensify/App/issues/44192#issuecomment-2191701019). The problems I mentioned above comes from the navigation refactor.

**Comment by situchan** [comment]
@kosmydel please let me know when ready

**Comment by kosmydel** [comment]
@situchan I think I found a solution that works with the latest changes, but I still need to consult it with my team.

[Here](https://github.com/Expensify/App/pull/41665/files/145513da1a8bb68b12e2535a878a6a308e7df7fa..9487f61b07329518a6a9303acd6000e273a6c51a) are changes that were made since the last review.

**Comment by situchan** [comment]
> Please test it thoroughly, as the logic has changed significantly due to concurrent mode and the recent navigation refactor.

We might need to put on hold until concurrent mode safely reaches production.
There's an attempt to [disable concurrent mode](https://github.com/Expensify/App/pull/44779) to unblock deploy.

**Comment by kosmydel** [comment]
@situchan do you think we can proceed with this PR?

**Comment by neil-marcellini** [comment]
Gentle bump @situchan

**Comment by situchan** [comment]
@kosmydel yes, I think we can proceed. Please mark as ready for review

**Comment by kosmydel** [comment]
@situchan I'm marking it as ready for review. 

FYI I've added a new test step, to test everything without Strict Mode, as this is what will be on production/staging.

File: src/pages/workspace/WorkspaceInitialPage.tsx
Comment (situchan) [inline src/pages/workspace/WorkspaceInitialPage.tsx:360]: I don't like using `USE_REACT_STRICT_MODE` flag here. It's not used anywhere yet except in `StrictModeWrapper`.

Comment (BrtqKr) [reply src/pages/workspace/WorkspaceInitialPage.tsx]: @situchan it wasn't necessary in the end, so we've removed it.

**Comment by situchan** [comment]
Lint failing

**Comment by BrtqKr** [comment]
@neil-marcellini @shawnborton would you mind taking a look 🥺

**Comment by shawnborton** [comment]
Happy to review from a design standpoint if there are any updated screenshots available.

**Comment by BrtqKr** [comment]
@shawnborton In this case, there would be no need for the review. This was only a small change in the code, it shouldn't affect anything in terms of design. 

@neil-marcellini gentle bump

**Review by neil-marcellini — APPROVED** [review]
The code looks fine, and I'll put my trust in @situchan's testing.

**Pull request merged**
